compiler/main/driver.cpp

```diff
@@ -584,6 +587,17 @@ static void setFastFlag(const ArgumentState* state, const char* unused) {
   specializeCCode = true;
 }
 
+static void setFloatOptFlag(const ArgumentState* state, const char* unused) {
```

Comment:
I'll add the comment, but unless we get one or two more like this, I think having the set function as I do here is a fine solution.